ota-plus-web/conf/application.conf

```diff
@@ -390,3 +390,5 @@ zipkin-trace-context {
 }
 
 play.modules.enabled  += "brave.play.module.ZipkinModule"
+
+omniture_source.url = ${?OMNITURE_SRC_URL}
```

Comment:
We should add a dummy fallback value for this, e.g.:
```
omniture_source.url = ""
omniture_source.url = ${?OMNITURE_SRC_URL}
```
Else `configuration.get[String]("omniture_source.url")` will throw an exception if `OMNITURE_SRC_URL` is not set.

Reply:
Done